Tests/BIntTests.swift

```diff
@@ -20,6 +20,12 @@ class BIntTests: XCTestCase {
         // Put teardown code here. This method is called after the invocation of each test method in the class.
         super.tearDown()
     }
+    
+//    func testSizeString()
```

Comment:
Remove or actually implement

Reply:
Please remove.  Not sure how this got in there.